### PR TITLE
Rename State's Root to Bundle

### PR DESCRIPTION
**Pull request by crosbymichael**
Signed-off-by: Michael Crosby <crosbymichael@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
LGTM

**Comment by mrunalp** [comment]
@vbatts @LK4D4 @vishh @philips ping

File: config.go [hunk 1]
```diff
@@ -65,6 +65,6 @@ type State struct {
 	ID string `json:"id"`
 	// Pid is the process id for the container's main process.
 	Pid int `json:"pid"`
-	// Root is the path to the container's bundle directory.
-	Root string `json:"root"`
+	// Bundle is the path to the container's bundle directory.
+	Bundle string `json:"bundle"`
```
Comment (vishh) [inline config.go]: nit: How about `BundlePath` since `Bundle` can be interpreted as the name of the bundle.

Comment (vbatts) [inline config.go]: :+1:

Comment (mrunalp) [inline config.go]: +1

**Comment by crosbymichael** [comment]
Updated

File: runtime.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ By providing a default location that container state is stored external applicat
 * **`version`** (string) Version of the OCI specification used when creating the container.
 * **`id`** (string) ID is the container's ID.
 * **`pid`** (int) Pid is the ID of the main process within the container.
-* **`root`** (string) Root is the path to the container's bundle directory.
+* **bundlePath** (string) BundlePath is the path to the container's bundle directory.
```
Comment (vbatts) [inline runtime.md]: make it **`pretty`** please

**Comment by vbatts** [comment]
one last nit, and LGTM

File: runtime.md (hunk 2 above)
Comment (crosbymichael) [inline runtime.md]: what do you mean? i don't see it?

Comment (vbatts) [inline runtime.md]: add backticks around "bundlepath"

**Comment by crosbymichael** [comment]
@vbatts done

**Comment by mrunalp** [comment]
LGTM. All comments have been addressed. Merging.

**Pull request merged**